x/attribute/types/msgs.go

```diff
@@ -9,14 +9,18 @@ import (
 )
 
 const (
-	TypeMsgAddAttribute    = "add_attribute"
-	TypeMsgDeleteAttribute = "delete_attribute"
+	TypeMsgAddAttribute             = "add_attribute"
+	TypeMsgUpdateAttribute          = "update_attribute"
+	TypeMsgDeleteAttribute          = "delete_attribute"
+	TypeMsgDeleteAttributeWithValue = "delete_attribute_with_value"
```

Comment:
Nit: It's called "Delete distinct" in all the other places.
```suggestion
	TypeMsgDeleteAttributeWithValue = "delete_distinct_attribute"
```
Or maybe `"delete_attribute_distinct"`.

Reply:
Thanks for this catch.  I had initial used `with_value` for everything.  Then did a refactor.